docs/docs/monitoring/analytics/getting-started-with-analytics.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: getting-started-with-analytics
+sidebar_label: Getting started
+title: Getting started with Analytics
+description:
+abstract: Rasa Pro Analytics is a data pipeline that extracts model, training,
+  and conversation data from the Rasa platform and streams the data to
+  your data warehouse for analysis.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import RasaProLabel from "@theme/RasaProLabel";
+import RasaProBanner from "@theme/RasaProBanner";
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+Analytics helps visualize and process Rasa assistant metrics in the
+tooling (BI tools, data warehouses) of your choice. Visualizations
+and analysis of the production assistant and its conversations allow
+you to assess ROI and improve the performance of the assistant over time.
+
+<div align="center">
+  <img
+    alt="Rasa Pro Analytics example dashboard"
+    src={useBaseUrl("/img/analytics/analytics-examples.png")}
+    width="100%"
+  />
+</div>
+
+Measuring a Rasa assistant's success and making strategic investments
+will lead to better business outcomes. It will also help you understand
+the needs of your end users and better serve them.
+
+<div align="center">
+  <img
+    alt="An overview of the components of Rasa Pro."
+    src={useBaseUrl("/img/rasa_pro_overview.png")}
```

Comment:
And this image

Reply:
I can draw something up, but ideally, we'd want something in the style of https://11591--rasa-community-docs.netlify.app/docs/rasa/img/introduction.png - who created that one?